api_proxy/proxy.go

```diff
@@ -0,0 +1,29 @@
+// Package main starts an App Engine API proxy on port 10001.
+// It is intended to allow migration of APIs from the Service Bridge to
+// publically available APIs and allow users & runtimes to migrate at a
+// later date.
+package main // import "github.com/GoogleCloudPlatform/appengine-sidecars-docker/api_proxy"
```

Comment:
If someone wanted to use golang-builder, it requires the import so I added it back again.